src/server/saved_objects/client/saved_objects_client.js

```diff
@@ -0,0 +1,116 @@
+import Boom from 'boom';
+import { get, omit } from 'lodash';
+
+import {
+  createFindQuery,
+  createFilterPath,
+  handleEsError,
+} from './lib';
+
+export class SavedObjectsClient {
+  constructor(kibanaIndex, callAdminCluster) {
+    this._kibanaIndex = kibanaIndex;
+    this._callAdminCluster = callAdminCluster;
+  }
+
+  async create(type, options = {}) {
+    const body = omit(options, 'id');
+
+    const response = await this._withKibanaIndex('index', {
+      type,
+      body
+    });
+
+    return Object.assign({
+      type: response._type,
+      id: response._id,
+      version: response._version
```

Comment:
Between the two, I am leaning towards the first. Looking at jsonapi.org it's something which they follow, specifically named `attributes`

From their docs:
```json
{
    "type": "comments",
    "id": "5",
    "attributes": {
      "body": "First!"
    },
    "relationships": {
      "author": {
        "data": { "type": "people", "id": "2" }
      }
    },
    "links": {
      "self": "http://example.com/comments/5"
    }
}
```